src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -306,3 +348,66 @@ export const SiteDashboard = (): JSX.Element => {
     </SiteViewLayout>
   )
 }
+
+const SiteLaunchDisplayCard = ({
+  siteName,
+  siteLaunchStatus,
+  isSiteLaunchLoading,
+  siteLaunchChecklistStepNumber,
+}: SiteLaunchDisplayCardProps): JSX.Element => {
```

Comment:
could we remove the `siteName/status/stepnumber` props? this is because of the (implicit) assumption that this component should always be rendered inside a `SiteLaunchContext` + `Route`.

this allows us to simplify this component and, if used without the appropriate context, throw an error

Reply:
> could we remove the siteName/status/stepnumber props?
Hmm I am lost here :(
these are variables that I intend to use later in this function, what exactly do you mean by removing the props ah? What does this got to do with being rendered inside a SiteLaunchContext + Route ya?

Reply:
these props are exposed on their respective hooks - `useParams`/`useSiteLaunchContext`. This eliminates the need for them to be passed as explicit props because **this component should not be used outside of the context**.